addons/web/static/src/webclient/clickbot/clickbot_loader.js

```diff
@@ -1,10 +1,10 @@
 /** @odoo-module alias=web.clickEverywhere **/
 
-import { loadJS } from "@web/core/assets";
+import { getBundle, loadBundle } from "@web/core/assets";
 import { registry } from "@web/core/registry";
 
 export default async function startClickEverywhere(xmlId, appsMenusOnly) {
-    await loadJS("web/static/src/webclient/clickbot/clickbot.js");
+    await loadBundle(await getBundle("web.assets_clickbot"));
```

Comment:
maybe not possible in 16.3, but in master you can directly call loadBundle with the name of the bundle